### PR TITLE
Fix the content according to the new support for flow description

### LINKED ISSUE #504: Fix the content according to the new support for flow description
see https://github.com/CloudSlang/cloud-slang/pull/545

### PR DESCRIPTION
**Pull request by gituser173**
Fixes #504
Signed-off-by: Claudiu Ban claudiu-laurian.ban@hpe.com

### CODE REVIEW

**Comment by gituser173** [comment]
@Bonczidai, @meshipeer, @mihaitusa, @SamMarkowitz please review :)

**Comment by SamMarkowitz** [comment]
I just took a quick glance. 

One thing I noticed in content/io/cloudslang/docker/swarm/clear_cluster.sl is that `Valid` was added as an input.

**Comment by gituser173** [comment]
@SamMarkowitz fixed that for all the files, thank you for your input, you can review

**Comment by mihaitusa** [comment]
I'm reviewing it :))

File: content/io/cloudslang/base/network/example/ping_hosts.sl [hunk 1]
```diff
@@ -6,26 +6,22 @@
...
-#   - username - optional - username to connect to email host - System Property: io.cloudslang.base.username
-#   - password - optional - password for the username to connect to email host - System Property: io.cloudslang.base.password
-#
-# Results:
-#   - SUCCESS - addressee will get an email with result
-#   - FAILURE - addressee will get an email with exception of operation
-#
+#!!
+#! @description: Pings addresses from input list and sends an email with results.
+#! @prerequisites: system property file with email properties
```
Comment (mihaitusa) [inline content/io/cloudslang/base/network/example/ping_hosts.sl]: @gituser173  Can be only "prerequisite" here the same as input, result, output? Maybe there are situation where more than one prerequisite is needed. WDYT ?

**Comment by mihaitusa** [comment]
@gituser173  It looks OK! I think it could be merged. :+1:

**Pull request merged**